test/spec/modules/atsAnalyticsAdapter_spec.js

```diff
@@ -2,7 +2,7 @@ import atsAnalyticsAdapter from '../../../modules/atsAnalyticsAdapter.js';
 import { expect } from 'chai';
 import adapterManager from 'src/adapterManager.js';
 import {server} from '../../mocks/xhr.js';
-import {browserIsChrome, browserIsEdge, browserIsFirefox, browserIsSafari} from '../../../modules/atsAnalyticsAdapter.js';
+import {checkUserBrowser, browserIsChrome, browserIsEdge, browserIsSafari} from '../../../modules/atsAnalyticsAdapter.js';
```

Comment:
Is there a reason for not testing browserIsFirefox()?

Reply:
Hi, yes, I didn't manage to mock InstallTrigger. We use this check to determine if user browser is firefox                     
(typeof InstallTrigger !== 'undefined'). 

https://developer.mozilla.org/en-US/docs/Web/API/InstallTrigger
https://stackoverflow.com/questions/9847580/how-to-detect-safari-chrome-ie-firefox-and-opera-browser/9851769